src/crackmes/hackthebox/find-the-easy-pass/bonus.md

```diff
@@ -0,0 +1,20 @@
+# Bonus
+
+In the previous pages, we discovered the right password
+but is it calculated or hard-coded.
+
+go to the address `0x00454084`
+
+![bonus-01](./img/bonus-01.png)
+
+Look the address `0x00454188`

```

Comment:
you should explain why you need to look at those addresses.
Why there and how did you get that address.

Reply:
I developed the why those addresses.